packages/realm/src/TypeHelpers.ts

```diff
@@ -110,6 +120,61 @@ export function mixedToBinding(realm: binding.Realm, value: unknown): binding.Mi
   }
 }
 
+//TODO NEED TO CHANGE ALSO WHAT HAPPENS ON JSI (NOT ONLY NODE)
```

Comment:
Don't forget!

Also, this comment seems misplaced here. It probably belongs on node.ts. But probably better to just implement it in jsi.ts and delete this comment rather than moving it.